hasher-matcher-actioner/hmalib/common/aws_dataclass.py

```diff
@@ -64,27 +64,27 @@ def py_to_aws(py_field: t.Any, in_type: t.Optional[t.Type[T]] = None) -> T:
             f"Expected {check_type} got {type(py_field)} ({py_field!r})"
         )
 
-    if in_type is int:  # N
+    if in_type == int:  # N
```

Comment:
Because of how github's comments are tied to the SHA, doing this would change the history for this PR. 

If I am not able to merge this in by Wednesday, I'll move to a dedicated PR and rebase.